ignite/contrib/handlers/param_scheduler.py

```diff
@@ -468,15 +468,32 @@ def __init__(self, schedulers, durations, save_history=False):
         self.schedulers = schedulers
         self.durations = durations
 
-        self.optimizer = self.schedulers[0].optimizer
-        if not (all(id(s.optimizer) == id(self.optimizer) for s in self.schedulers)):
+        param_schedulers = [s for s in self.schedulers if isinstance(s, ParamScheduler)]
+        param_group_schedulers = [s for s in self.schedulers if isinstance(s, ParamGroupScheduler)]
+
+        param_optimizers = [s.optimizer for s in param_schedulers]
+        param_group_optimizers = [s.optimizer for sc in param_group_schedulers for s in sc.schedulers]
+        optimizer = list(set(param_optimizers + param_group_optimizers))
+        if len(optimizer) != 1:
             raise ValueError("schedulers should be related to same optimizer")
 
+        self.optimizer = optimizer[0]
+
+        param_names = [s.param_name for s in param_schedulers]
+        param_group_names = [n for sc in param_group_schedulers for n in sc.names]
```

Comment:
I think this can be incorrect: `sc.names` are not related to `param_name`. See, example in docstring :
```python
scheduler1 = LinearCyclicalScheduler(optimizer, 'lr', 1e-7, 1e-5, len(train_loader), param_group_index=0)
scheduler2 = CosineAnnealingScheduler(optimizer, 'lr', 1e-5, 1e-3, len(train_loader), param_group_index=1)
lr_schedulers = [scheduler1, scheduler2]
names = ["lr (base)", "lr (fc)"]
scheduler = ParamGroupScheduler(schedulers=lr_schedulers, names=names)
```

Reply:
Oh yes, you are right !! Thank you !